wheel/chia_rs.pyi

```diff
@@ -91,11 +91,8 @@ class G1Element:
     def get_fingerprint(self) -> int: ...
     def pair(self, other: G2Element) -> GTElement: ...
     @staticmethod
-    def from_bytes_unchecked(b: bytes) -> G1Element: ...
-    @staticmethod
     def generator() -> G1Element: ...
     def __str__(self) -> str: ...
-    def __repr__(self) -> str: ...
```

Comment:
are all the `__repr__()` and `from_bytes_unchecked()` that you're removing duplicates?

Reply:
yes